index.html

```diff
@@ -1264,11 +1264,42 @@
       </section>
 
       <section>
-        <h3>Storage Location of Claims</h3>
+        <h3>Storage Providers and Data Mining</h3>
 
-        <p class="issue">
-Storage location of claims may mine data or forward data.
+        <p>
+When a holder receives a claim from an issuer, it is highly likely that the
```

Comment:
No need to say "highly likely", just "the claim may need to be stored somewhere"